backend/benefit/applications/api/v1/serializers.py

```diff
@@ -75,10 +77,39 @@ class Meta:
         }
 
 
+class AttachmentField(FileField):
+    def to_representation(self, value):
+        if not value:
+            return None
+
+        url_pattern_name = "v1:applicant-application-download-attachment"
+        request = self.context.get("request")
+        if (
+            request
+            and (user := get_request_user_from_context(self))
+            and user.is_handler()
```

Comment:
Using `:=` here is unnecessary and makes the logic slightly brittle i.e. easier to break.
It's ok to leave this code as it is.